Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,38 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string name;
```

Comment:
Don't declare all variables in one place- it disrupts readability, because you will have to scroll up and down all the time.

What you should be doing instead is declaring variables right next to where you use them.